packages/strapi-plugin-settings-manager/package.json

```diff
@@ -48,7 +48,7 @@
     "plop": "^1.5.0",
     "prettier": "^1.5.3",
     "rimraf": "^2.5.4",
-    "strapi-helper-plugin": "3.0.0-alpha.5.5",
+    "strapi-helper-plugin": "file:../strapi-helper-plugin",
```

Comment:
Please run `npm run clean` before pushing to avoid relative path.

Reply:
`clean` script is launch on pre-commit (see in package.json) - we have to know why it failed